src/hotspot/share/utilities/growableArray.hpp

```diff
@@ -259,8 +259,8 @@ class GrowableArrayView : public GrowableArrayBase {
 
   // Remove all elements in the range [start - end). The order is preserved.
   void remove_range(int start, int end) {
-    assert(0 <= start, "illegal index");
-    assert(start < end && end <= _len, "erase called with invalid range");
+    assert(0 <= start, "illegal start index %d", start);
+    assert(start < end && end <= _len, "erase called with invalid range %d, %d", start, end);
```

Comment:
I think you'd want to see `_len` here too - suggestion:

"erase called with invalid range (%d, %d) for length %d", start, end, _len

Reply:
Hi David, I agree !